packages/admin/cms-admin/src/builds/PublishButton.tsx

```diff
@@ -1,80 +1,30 @@
-import { LocalErrorScopeApolloContext } from "@comet/admin";
-import { Button, CircularProgress, Theme } from "@mui/material";
-import { green, red } from "@mui/material/colors";
+import { Button, Theme } from "@mui/material";
 import createStyles from "@mui/styles/createStyles";
 import makeStyles from "@mui/styles/makeStyles";
-import clsx from "clsx";
 import * as React from "react";
 import { FormattedMessage } from "react-intl";
 
-import { GQLCreateBuildMutation, namedOperations } from "../graphql.generated";
-
-const createBuildMutation = gql`
-    mutation CreateBuild {
-        createBuild
-    }
-`;
+import { StartBuildsDialog } from "./StartBuildsDialog";
 
 export const PublishButton: React.FunctionComponent = () => {
     const classes = useStyles();
-    const [startBuild, { loading, error, data }] = useMutation<GQLCreateBuildMutation>(createBuildMutation);
-    const [hasStarted, setHasStarted] = React.useState(false);
-    const [hasError, setHasError] = React.useState(false);
-
-    React.useEffect(() => {
-        if (error) {
-            setHasError(true);
-
-            const timeout = window.setTimeout(() => {
-                setHasError(false);
-            }, 2000);
-
-            return () => {
-                window.clearTimeout(timeout);
-            };
-        }
-    }, [error]);
-
-    React.useEffect(() => {
-        if (data?.createBuild === true) {
-            setHasStarted(true);
-
-            const timeout = window.setTimeout(() => {
-                setHasStarted(false);
-            }, 2000);
-
-            return () => {
-                window.clearTimeout(timeout);
-            };
-        }
-    }, [data]);
+    const [isStartBuildsDialogOpen, setIsStartBuildsDialogOpen] = React.useState(false);
 
     const handlePublishClick = () => {
-        startBuild({
-            refetchQueries: [namedOperations.Query.Builds],
-            context: LocalErrorScopeApolloContext,
-        });
+        setIsStartBuildsDialogOpen(true);
     };
 
     return (
         <div className={classes.wrapper}>
-            <Button
-                variant="contained"
-                color="primary"
-                className={clsx({
-                    [classes.buttonSuccess]: hasStarted,
-                    [classes.buttonError]: hasError,
-                })}
-                disabled={loading}
-                onClick={handlePublishClick}
-                fullWidth
-            >
-                {hasStarted && <FormattedMessage id="comet.pages.publisher.buildStarted" defaultMessage="Build started" />}
-                {hasError && <FormattedMessage id="comet.pages.publisher.buildStartFailed" defaultMessage="Starting build failed" />}
-                {!hasStarted && !hasError && <FormattedMessage id="comet.pages.publisher.build" defaultMessage="Start build" />}
+            <Button variant="contained" color="primary" onClick={handlePublishClick} disabled={process.env.NODE_ENV === "development"} fullWidth>
```

Comment:
Should be disable the button when a build is already running?

Reply:
I would not do this as you could start a build for scope A and after that you want to start a build for scope B. We could disable the selection for already running builds, but I think that is an overkill, as the api does not trigger a new build if one is already running for that scope